src/Illuminate/Support/Facades/Vite.php

```diff
@@ -2,12 +2,18 @@
 
 namespace Illuminate\Support\Facades;
 
+use Illuminate\Foundation\Vite as Factory;
```

Comment:
Can we revert this, as it is inconsistent with the other Facades that aren't backed by a contract / existing binding i.e. `RateLimiter` and `ParallelTesting`